pkg/network/ovn_kubernetes.go

```diff
@@ -341,7 +342,16 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 		objs = k8s.RemoveObjByGroupKindName(objs, "apps", "DaemonSet", "openshift-ovn-kubernetes", "ovnkube-upgrades-prepuller")
 	}
 
-	return objs, nil
+	if bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.Enabled && bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.Route == "" {
+		k8s.UpdateObjByGroupKindName(objs, "apps", "DaemonSet", "openshift-ovn-kubernetes", "ovnkube-node", func(o *uns.Unstructured) {
+			anno := o.GetAnnotations()
```

Comment:
I think `GetAnnatations` could return `nil`, we might need to add a check if it does

Reply:
Added a conditional check on `anno == nil`.